plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -51,6 +99,423 @@ def get_ilo_sessions(self):
         result["ret"] = True
         return result
 
+    def get_systems_data(self):
+        response = self.get_request(self.root_uri + self.systems_uri)
+        if not response["ret"]:
+            return response
+
+        return {
+            "ret": True,
+            "msg": response
+        }
+
+    def get_managers_data(self):
+        response = self.get_request(self.root_uri + self.manager_uri)
+        if not response["ret"]:
+            return response
+
+        return {
+            "ret": True,
+            "msg": response
+        }
+
+    def get_service_bios_attributes(self):
```

Comment:
Getting BIOS attributes is standard Redfish functionality; we have this today as "get_bios_attributes". Why is this needed in an iLO-specific module?

Reply:
get_service_bios_attributes() queries a different a API then the get_bios_attributes() function
get_service_bios_attributes API - oem/hpe/service/settings/ - This provides the service options available in the server.

Reply:
I see that now; as long as this goes off into OEM, then I agree this can stay here.

Reply:
Thank you, closing this conversation thread.